test/e2e-tests.sh

```diff
@@ -14,15 +14,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export GO111MODULE=on
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 
+# Namespace used for tests
+readonly TEST_NAMESPACE="operator-tests"
+
+function tekton_setup() {
+  header "Installing Tekton Operator"
+  kubectl create namespace $TEST_NAMESPACE
+  kubectl apply -n $TEST_NAMESPACE -f deploy/crds/
+  ko apply -n $TEST_NAMESPACE -f config/
+  wait_until_pods_running $TEST_NAMESPACE
+}
+
+function tekton_teardown() {
+  header "Removing Tekton Operator"
+  kubectl delete --ignore-not-found Config cluster
+  ko apply -n $TEST_NAMESPACE -f config/
+  kubectl delete -n $TEST_NAMESPACE -f deploy/crds/
```

Comment:
why do we need `-n $TEST_NAMESPACE` while applying CRDs as CRDs are clusterwide

Reply:
Again, I have the link files of crds under config/, the crds will be removed if I run kubectl delete -n $TEST_NAMESPACE -f config/. I will change the tear_down as well.

Reply:
> @nikhil-thomas per ./test/e2e-tests.sh, this is an existing old issue with this script all across knative and tekton projects. We have to set an env var PROJECT_ID or --gcp-project to run the script, no matter we use google cloud or not. I will get to it to see, if we can resolve it in a wide range.

it set the env var PROJECT_ID, but then i need `gcloud` binary. In addtion, I have to provide a valid PROJECT_ID not any random value. :(

is that how it works at present

Reply:
it  makes more sense now :+1: